pyproject.toml

```diff
@@ -55,6 +55,9 @@ crewai = [
 google-genai = [
     "google-genai>=1.2.0",
 ]
+audio = [
```

Comment:
we would need to document that if you want to use audio you need to add audio extra, I think raising the error at runtime in the adapter is not sufficient.

Reply:
Also we would need to add audio to the test group